universal-login-sdk/lib/api/FutureWalletFactory.ts

```diff
@@ -24,7 +24,7 @@ export interface Deployment {
   waitToBeSuccess: () => Promise<DeployedWallet>;
 }
 
-export interface FutureWallet extends CounterfactualWallet {
+export interface FutureWallet extends SerializableFutureWallet {
```

Comment:
Why do we need to extend?

Reply:
We don't, but `FutureWallet` has the same data fields as `SerializableFutureWallet`. `extend` is an alternative to just repeating those field in there. Do you think it would be better to repeat them instead?